### ISSUE TITLE
General Questions

### ISSUE THREAD

**Issue by ManuelBuecherl** [taarskog/crm-powerbi-viewer#58]
Hi Trond,

i hope this is the right way to place my request regarding the PowerBIViewer.

I'd like to implement the tool at a Client, but he has following questions:

1. Does the tool also run under v9.1 of D365 (CE)? I've found contrary information about this.
2. Does every User need the PowerBI Pro License to be able to display the embedded Reports?
3. Is it only possible to display Content of Workspaces or also content of "Apps"? Out client is providing his Reports via apps, since they are having a "Premium capacity" License Model

Best Regards
Manuel

**Comment by taarskog** [comment]
Hi Manuel,

1. It runs fine using the "old" UI. I would not recommend using the current version with the new unified interface. If I decide to continue updating the solution I will as one of my first tasks add support for Unified Interface (read more below).
2. See my comment and link to official statement from Microsoft at #43. Note the part on Premium...
3. Apps should work fine. Suggest you verify this by previewing the apps on the solution configuration page.

Regarding the future of crm-powerbi-viewer I recommend reading the red note at the top of http://crm-powerbi-viewer.heiigjen.com/. If you do require embedding visuals or handle events my solution may be the way to go. If those are not required you may be better off using the soon to be released (I hope) out-of-the-box support.

The PowerBI team has also released support for a new parameter "&autoAuth=true" which basically handles the authentication (and original reason for me creating crm-powerbi-viewer). Ulrik has written about it on his site - https://crmchartguy.com/2019/02/18/embed-power-bi-visuals-in-dynamics-365-with-secure-embed/.

So to sum up the need for crm-powerbi-viewer may soon come to an end. It all depend on what functionality and support Microsoft ends up providing out of the box. No further development will be done to my solution the next couple of months while we wait for the answer (not even possible until Microsoft comes with official documentation on developing custom controls on the unified interface).

Hope this helps,
Trond
